etcd-procedures/docs/add_etcd_node.adoc

```diff
@@ -0,0 +1,93 @@
+== Introduction
+
+This procedure will cover the following points in order to scale up an existing etcd Cluster
+
+- Install etcd as a Docker container in RHEL or RHEL Atomic
+- Add etcd Nodes to existing etcd Cluster
+
+== Requirements
+
+- A working etcd Cluster with at least 1 Node
+- DNS working
+- Access to registry.access.redhat.com for image pulling
+- SSH access from the existing etcd Nodes to new etcd Nodes
+
+
+=== Install etcd as a Docker container
+
+NOTE: Check if etcd needs to be installed in order to perform OCP upgrades
+
+==== From one of the existing etcd Nodes, get the etcd container image version we need to run
```

Comment:
Would you be able to provide context into what the commands that are performing?